src/Form/DataTransformer/CommitteeToIdTransformer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppBundle\Form\DataTransformer;
+
+use AppBundle\Entity\Committee;
+use AppBundle\Repository\CommitteeRepository;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+class CommitteeToIdTransformer implements DataTransformerInterface
```

Comment:
Do you try use `ModelToIdPropertyTransformer` of Sonata?

Reply:
:carousel_horse: how about no?